sdk/containerregistry/azure-containerregistry/tests/testcase.py

```diff
@@ -191,18 +143,13 @@ def get_token(self, *args):
 
 class ContainerRegistryTestClass(AzureTestCase):
     def __init__(self, method_name):
-        super(ContainerRegistryTestClass, self).__init__(
-            method_name,
-            recording_processors=[
```

Comment:
For my own clarity. We are using the default set of recording_processors from `azure_testcase`, which would be just

```
    def _get_recording_processors(self):
        return [
            GeneralNameReplacer()
            AuthenticationMetadataFilter(),
            OAuthRequestResponsesFilter(),
            RequestUrlNormalizer(),
        ]
```

Seems fine to me!